source/databricks/settlement_report/settlement_report_job/entry_point.py

```diff
@@ -27,23 +28,38 @@
     parse_command_line_arguments,
 )
 from settlement_report_job.infrastructure.spark_initializor import initialize_spark
+from settlement_report_job.domain.task_type import TaskType
 
 
 # The start() method should only have its name updated in correspondence with the
 # wheels entry point for it. Further the method must remain parameterless because
 # it will be called from the entry point when deployed.
-def start() -> None:
+def start_hourly_time_series() -> None:
+    _start_task(TaskType.HOURLY_TIME_SERIES)
```

Comment:
I'm in favor of the open-closed principle and would like to avoid the if-else handling below.
This can be achieved by something similar to this:
```suggestion
    _start_task(lambda spark, args: report_generator.execute_hourly_time_series(spark, args))
```